src/app/rundown/components/segment/segment.component.html

```diff
@@ -1,4 +1,44 @@
-<div class="segment" [ngClass]="{'segment-on-air': segment.isOnAir}">
+<div class="c-segment" [class.c-segment--on-air]="segment.isOnAir" [class.c-segment--set-as-next]="segment.isNext">
+  <div class="c-segment__header">
+    <div class="c-segment__header-toolbar">
+      <span>123</span>
+      <span style="margin-left: 50px;">00:30</span>
+    </div>
+    <div class="c-segment__title">
+      {{ segment.name }}
+    </div>
+
+  </div>
+
+  <div class="c-segment__controls">
+    <div class="c-segment__controls-toolbar">Hello</div>
+    <div class="c-segment__piece-layers">
+      <div class="c-segment__piece-layer" *ngFor="let pieceLayer of pieceLayers">
+        {{ pieceLayer }}
+      </div>
+    </div>
+  </div>
+
+  <!-- TODO: Should this be a separate component, such that we can easily swap it out. E.g. if we want to follow playhead or show full overview. -->
+  <div class="c-segment__timeline">
+    <sofie-timeline [ngClass]="'c-segment__timeline-markers'" [time]="timeReference" [pixelsPerSecond]="30"></sofie-timeline>
+    <div class="c-segment__timeline-parts">
+      <div class="c-segment__timeline-parts-wrapper">
```

Comment:
Do you really need both these divs?
The first does literally nothing but add padding.

Reply:
`.c-segment__timeline-parts-wrapper` was intended for organizing the part content, where `.-segment__timeline-parts` was the scroll-container.

Since we don't scroll at the moment, the wrapper div is removed.